src/main/java/edu/indiana/dlib/amppd/service/impl/BatchServiceImpl.java

```diff
@@ -438,7 +444,7 @@ private Item getItem(Collection collection, String itemName, String itemDescript
 			for(Item i : items) {
 				if(!externalSource.isBlank() && !externalItemId.isBlank()) 
 				{
-					if(i.getExternalIds().containsKey(externalItemId))
+					if(i.getExternalIds() != null && i.getExternalIds().containsKey(externalItemId))
 					{
 						found = true;
 						item = i;
```

Comment:
Do we have a unit test to cover the case where Maria found the bug? If not, could you add one? thanks!